addons/binding/org.openhab.binding.networkcamera/README.md

```diff
@@ -0,0 +1,83 @@
+# Network Camera Binding
+
+This binding integrates large number of network cameras which can send image to FTP server when motion or sound is detected. Binding acts as a FTP server. Images stored to FTP server are not saved to the file system, therefore binding shouldn't cause any problems on flash based openHAB installations.
+
+
+## Supported Things
+
+This binding supports ```motiondetection``` Thing. Every camera is identified by FTP user name. Therefore, every camera should use unique user name to login FTP server.
+
+## Discovery
+
+Automatic discovery is not supported.
+
+## Binding Configuration
+
+Bindings FTP server listening 2121 TCP port by default, but port can be configured. Also idle timeout can be configured.
+
+## Channels
+
+This binding currently supports following channels:
+
+| Channel Type ID | Item Type    | Description                                                                            |
+|-----------------|--------------|----------------------------------------------------------------------------------------|
+| image           | Image        | Image received from network camera.                                                    |
+| motion          | Switch       | Motion detection sensor state. Updated to ON state when image is received from camera. |
+
+
+### Trigger Channels
+
+| Channel Type ID | Options                | Description                                        |
+|-----------------|------------------------|----------------------------------------------------|
+| motion-trigger  | MOTION_DETECTED        | Triggered when image received from network camera. |
```

Comment:
see above: I a rule you can also check if the image channel has received and update, so that one channel should be sufficient.

Reply:
Motion trigger channel is useful, if someone don't want to have image channel e.g. for private reaseon.